tests/scattering1d/test_numpy_backend_1d.py

```diff
@@ -0,0 +1,117 @@
+import pytest
```

Comment:
are these tests from this pr or from the tests refactoring pr? I don't see why the tests would change with this refactoring

Reply:
This is from the test refactoring PR. We should merge those as soon as possible.

Reply:
hmm,  i believe this is merged, so I'm surprised to see the diff below